components/textbox.jsx

```diff
@@ -174,6 +174,12 @@ export default class Textbox extends React.Component {
         const hasText = this.props.value && this.props.value.length > 0;
 
         let editHeader;
+        let helpTextClass = '';
+
+        if (this.props.value.length > this.props.characterLimit) {
```

Comment:
Can you change this to `this.props.value && this.props.value.length > this.props.characterLimit`? A bit of defensive code in case `this.props.value` is undefined